agent/taskresource/cgroup/cgroup.go

```diff
@@ -345,3 +348,13 @@ func (cgroup *CgroupResource) GetCgroupMountPath() string {
 	defer cgroup.lock.RUnlock()
 	return cgroup.cgroupMountPath
 }
+
+// Synchronize fills the resource fileds
+func (cgroup *CgroupResource) Synchronize(resourceFields *taskresource.ResourceFields) {
```

Comment:
This is just doing initialization. Can you rename `Synchronize` to `Initialize` everywhere?